dbt_project.yml

```diff
@@ -23,6 +23,11 @@ clean-targets:         # directories to be removed by `dbt clean`
   - "target"
   - "dbt_packages"
 
+flags:
+  require_explicit_package_overrides_for_builtin_materializations: false  # Allow packages to override materializations
+
```

Comment:
same question. what is this for ? configurations are usually set at the start of the project and they dont change a lot.

Reply:
After I put the above suggestion, this popped up : "Starting from dbt 1.8, users must explicitly allow packages to override materializations. 
Elementary requires this ability to support collection of samples and failed row count for dbt tests.
Please add the following flag to dbt_project.yml to allow it:

flags:
  require_explicit_package_overrides_for_builtin_materializations: false"